doc/node.1

```diff
@@ -199,6 +205,9 @@ Set the
 .Ar host:port
 to be used when the inspector is activated.
 .
+.It Fl -inspect-publish-uid=stderr,http
+Specify how the inspector WebSocket URL is exposed.
+.
```

Comment:
Can we list the valid values for this option here?

Reply:
Done.